### PR TITLE
Prepare the 2.1.61 release.

### LINKED ISSUE #1552: Release 2.1.61
On the docket:
+ [x]  Merge packages for --venv-site-packages-copies. #1557

### PR DESCRIPTION
**Pull request by jsirois**
Closes #1552

### CODE REVIEW

**Review by asherf — APPROVED** [review]

**Comment by jsirois** [comment]
This will need a maintainer sign-off.

**Review by kaos — APPROVED** [review]

**Comment by jsirois** [comment]
Thanks @kaos.

**Pull request merged**
